lib/v2/issuehunt/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'issuehunt.io': {
+        _name: 'Issue Hunt',
+        '.': [
+            {
+                title: '悬赏',
```

Comment:
```suggestion
                title: '项目悬赏',
```
The title here should be the same as H3 header of the docs